app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -252,6 +252,7 @@ def parse_portgroups_internal(object, props)
       :allow_promiscuous => allow_promiscuous,
       :forged_transmits  => forged_transmits,
       :mac_changes       => mac_changes,
+      :tag               => default_port_config&.vlan&.try(:vlanId)
```

Comment:
`.try()` will successfully handle the possible `NoMethodError` but it'd be a little more obvious what we're doing if we checked on the class of the object like:

```
vlan_spec = default_port_config&.vlan
tag = vlan_spec.vlanId if vlan_spec&.kind_of?(RbVmomi::VIM::VmwareDistributedVirtualSwitchVlanIdSpec)
```